components/teams-service/storage/conformance/conformance_test.go

```diff
@@ -172,20 +171,19 @@ func testGetTeamsForUser(ctx context.Context, t *testing.T, s storage.Storage) {
 
 	initialUsers := []string{"user-for-team-fetch", "two", "three"}
 
-	team, err := s.AddUsers(ctx, id, initialUsers)
+	_, err = s.AddUsers(ctx, id, initialUsers)
 	require.NoError(t, err, "setup: failed to add users to team-1-with-user")
-	team2, err := s.AddUsers(ctx, id2, initialUsers)
+	_, err = s.AddUsers(ctx, id2, initialUsers)
 	require.NoError(t, err, "setup: failed to add users to team-2-with-user")
 	_, err = s.AddUsers(ctx, idWithout, []string{"two", "three"})
 	require.NoError(t, err, "setup: failed to add users to test team-without-user")
 
 	returnedTeams, err := s.GetTeamsForUser(ctx, "user-for-team-fetch")
 	require.NoError(t, err)
-	expectedFetchedTeams := []storage.Team{team, team2}
-	ids := []string{returnedTeams[0].ID, returnedTeams[1].ID}
-	expectedIDs := []string{expectedFetchedTeams[0].ID, expectedFetchedTeams[1].ID}
+	expectedFetchedTeamIDs := []string{id, id2}
```

Comment:
since we no longer have team GUIDs to worry about and we can just use the user-created ID, had the opportunity to simplify some test code